sdks/typescript/src/clients/worker/worker.ts

```diff
@@ -144,7 +145,7 @@ export class V0Worker {
   async registerWorkflow(initWorkflow: Workflow) {
     const workflow: Workflow = {
       ...initWorkflow,
-      id: (this.client.config.namespace + initWorkflow.id).toLowerCase(),
+      id: withNamespace(initWorkflow.id, this.client.config.namespace).toLowerCase(),
```

Comment:
same here 😅  this is a bit confusing - I guess it'd be a breaking change to fix this maybe? this feels like the kind of thing we'll try to reread in a few months and not understand though